clang/docs/ReleaseNotes.rst

```diff
@@ -316,6 +316,7 @@ Bug Fixes in This Version
 - Fixed a modules crash where exception specifications were not propagated properly (#GH121245, relanded in #GH129982)
 - Fixed a problematic case with recursive deserialization within ``FinishedDeserializing()`` where
   ``PassInterestingDeclsToConsumer()`` was called before the declarations were safe to be passed. (#GH129982)
+- Fixed a module crash with `-dump-deserialized-decls`
```

Comment:
NIT: is it `-Xclang=-dump-deserialized-decls`? Users might be confused if they decide to try it out simply because it's mentioned in the release notes.